frappe/core/doctype/user/user.py

```diff
@@ -305,12 +305,10 @@ def get_other_system_managers(self):
 			.from_(user_role_doctype)
 			.select(user_doctype.name)
 			.where(user_role_doctype.role == "System Manager")
-			.where(user_doctype.docstatus < 2)
 			.where(user_doctype.enabled == 1)
 			.where(user_role_doctype.parent == user_doctype.name)
 			.where(user_role_doctype.parent.notin(["Administrator", self.name]))
 			.limit(1)
-			.distinct()
```

Comment:
This is required in some cases because this query is implicitly joining the child table. 

In this case since child table can't have multiple rows with same role it should be fine.

Reply:
it's also using limit 1 so distinct didn't really make sense :P